flight/src/tests.rs

```diff
@@ -0,0 +1,107 @@
+                )
+                .unwrap(),
+                stream_type: StreamIdType::Model as u8,
+                controller: "did:key:test4".to_string(),
+                dimensions: vec![],
+            },
+            previous: vec![],
+            data: vec![7, 8, 9],
+        }),
+    ];
+    // Convert events to RecordBatch
+    let record_batch = conclusion_events_to_record_batch(&events).unwrap();
+
+    // Convert RecordBatch to string
+    let formatted = pretty_format_batches(&[record_batch.clone()])
+        .unwrap()
+        .to_string();
+
+    // Use expect_test to validate the output
+    expect![[r#"
+        +------------+--------------------------------------------------------------------------+-------------+---------------+--------------------------------------------------------------------------+--------+----------------------------------------------------------------------------+
+        | event_type | stream_cid                                                               | stream_type | controller    | event_cid                                                                | data   | previous                                                                   |
+        +------------+--------------------------------------------------------------------------+-------------+---------------+--------------------------------------------------------------------------+--------+----------------------------------------------------------------------------+
+        | 0          | 01001220ba28cb0268f4f50cca99d952b610bfa2823941bfb12452ccf1e330725d96ae0d | 2           | did:key:test1 | 010012200b9691e769ef527bd51427fc745c8bfb8e89271e054548a780ddeed7817cbe0a | 010203 | []                                                                         |
+        | 0          | 01001220ba28cb0268f4f50cca99d952b610bfa2823941bfb12452ccf1e330725d96ae0d | 2           | did:key:test2 | 010012200b9691e769ef527bd51427fc745c8bfb8e89271e054548a780ddeed7817cbe0a | 040506 | [01001220ba28cb0268f4f50cca99d952b610bfa2823941bfb12452ccf1e330725d96ae0d] |
+        | 1          | 01001220ba28cb0268f4f50cca99d952b610bfa2823941bfb12452ccf1e330725d96ae0d | 2           | did:key:test3 | 010012200b9691e769ef527bd51427fc745c8bfb8e89271e054548a780ddeed7817cbe0a |        |                                                                            |
+        | 0          | 01001220ba28cb0268f4f50cca99d952b610bfa2823941bfb12452ccf1e330725d96ae0d | 2           | did:key:test4 | 010012200b9691e769ef527bd51427fc745c8bfb8e89271e054548a780ddeed7817cbe0a | 070809 | []                                                                         |
+        +------------+--------------------------------------------------------------------------+-------------+---------------+--------------------------------------------------------------------------+--------+----------------------------------------------------------------------------+"#]].assert_eq(&formatted);
+}
```

Comment:
Its hard to audit this when the CID are in hex instead of their common string representation. In my branch I added a small function to convert the cid bytes to a string [here](https://github.com/ceramicnetwork/rust-ceramic/pull/500/files#diff-eb701a6fc846dc0a4b096209cdf1b5b1a75028561fcc2e3c6629862757eec348R188-R233). Can you copy that function and use it here on the output? You can see how I use it [here](https://github.com/ceramicnetwork/rust-ceramic/pull/500/files#diff-eb701a6fc846dc0a4b096209cdf1b5b1a75028561fcc2e3c6629862757eec348R275-R284)

Reply:
not sure if it belongs in this test file. It seems like a lot of code for what we want to do, instead I added a method which converts cid bytes to string, for event_cid and stream_cid columns, which does the same thing
Open to push back